src/Components/ArtworkFilter/ArtworkFiltersQuick/FeaturedKeywordsFilterQuick.tsx

```diff
@@ -16,6 +16,10 @@ export const FeaturedKeywordsFilterQuick: FC<FeaturedKeywordsFilterQuickProps> =
   if (!props.featuredKeywords || !props.featuredKeywords.length) return null
   return (
     <>
+      <Spacer y={2} />
+      <Box width="1px" bg="black30" />
+      <Spacer y={2} />
```

Comment:
It'd be better to insert this in the parent component: https://github.com/artsy/force/blob/bd89cb774fc63423f06146bbddb9e03194184927/src/Components/ArtworkFilter/ArtworkFiltersQuick.tsx#L44

Rather than having space hanging off this one.

Reply:
Even if that requires the parent to do that conditionally, using the same logic as above?

Reply:
Oh... I didn't see that. Yeah, that's annoying. This is fine!

Reply:
Technically you could get away with out the empty space hanging off, but it doesn't matter if this thing isn't going to be used anywhere else.